### ISSUE TITLE
Video removed by uploader remains in download queue

### ISSUE THREAD

**Issue by sideloading** [w0d4/yt-backup#34]
**Describe the bug**
A video which has been removed by the channel uploader stays in the download queue.

**To Reproduce**
videoID: `Fc1CV6MWAfw` 

**Expected behavior**
Mark the video as unavailable, remove from download queue

**Debug log**
```
2020-06-06 12:09:06,257 - yt-backup - INFO - Video Fc1CV6MWAfw - (ASMR)  EATING GUMMIES AND FRUITS // EXTREME CHEWING AND MOUTH SOUNDS is not yet downloaded. Downloading now.
2020-06-06 12:09:06,271 - yt-backup - DEBUG - youtube-dl command is: /usr/local/bin/youtube-dl --continue  -4 --download-archive archive.list --output /tmp/"AriLoves ASMR [UCPQebUctwJpUpkzakg5Dc4A]"/"%(upload_date)s %(title).100s %(height)sp %(id)s.%(ext)s" --ignore-config --ignore-errors --merge-output-format mkv  --no-overwrites --format "(bestvideo+bestaudio/best)" --user-agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.122 Safari/537.36" --write-info-json --add-metadata --write-thumbnail --prefer-ffmpeg --merge-output-format mkv --no-overwrites --external-downloader aria2c --external-downloader-args '-j 8 -s 8 -x 8' https://youtu.be/Fc1CV6MWAfw
2020-06-06 12:09:08,476 - yt-backup - DEBUG - b'[youtube] Fc1CV6MWAfw: Downloading webpage\n'
2020-06-06 12:09:08,476 - yt-backup - DEBUG - b'ERROR: Fc1CV6MWAfw: YouTube said: This video has been removed by the uploader\nSorry about that.\n'
2020-06-06 12:09:08,476 - yt-backup - DEBUG - youtube-dl stdout: [youtube] Fc1CV6MWAfw: Downloading webpage
2020-06-06 12:09:08,476 - yt-backup - DEBUG - Current line: [youtube] Fc1CV6MWAfw: Downloading webpage
2020-06-06 12:09:08,476 - yt-backup - DEBUG - Current line: [youtube] Fc1CV6MWAfw: Downloading webpage
```

**Additional context**

**Comment by w0d4** [comment]
Fixed in 9d486de

**Comment by sideloading** [comment]
Thank you :)
